scripts/toggle-codec-logs.sh

```diff
@@ -45,6 +45,10 @@ case "$ACTION" in
         adb shell setprop log.tag.BufferQueueProducer D
         adb shell setprop log.tag.ReflectedParamUpdater D
         adb shell setprop log.tag.hw-BpHwBinder D
+        adb shell setprop log.tag.ACodec D
```

Comment:
I found this when running on a pixel device, so just added them here, not related to this PR